Javascript/index.js

```diff
@@ -0,0 +1,52 @@
+const ham = document.getElementById('hamburger');
+const men = document.getElementById('menu-items');
+
+ham.addEventListener('click', () => {
+  men.classList.toggle('bgShow');
+});
+
+men.addEventListener('click', () => {
+  men.classList.remove('bgShow');
+});
+
+// // popup
+
+// const projects = [
+//   {
+//     title: 'About the project #1',
+//     description: 'Descript',
+//     tech: ['HTML', 'JavaScript', 'Css', 'Ruby'],
+//     imgUrl: './assets/images/bg.jpg',
+//   },
+//   {
+//     title: 'About the project #2',
+//     description: 'Description of the project, date and languages',
+//     tech: ['HTML', 'JavaScript', 'Css', 'Ruby'],
+//     imgUrl: './assets/images/bg.jpg',
+//   },
+//   {
+//     title: 'About the project #3',
+//     description: 'Description of the project, date and languages',
+//     tech: ['HTML', 'JavaScript', 'Css', 'Ruby'],
+//     imgUrl: './assets/images/bg.jpg',
+//   },
+//   {
+//     title: 'About the project #4',
+//     description: 'Description of the project, date and languages',
+//     tech: ['HTML', 'JavaScript', 'Css', 'Ruby'],
+//     imgUrl: './assets/images/bg.jpg',
+//   },
+// ];
```

Comment:
Kindly delete commented code, it would be professional to make use of git history instead.

Reply:
Well resolved